docs/how_to_guides/creating_and_editing_expectations/how_to_create_custom_expectations_for_sqlalchemy.rst

```diff
@@ -3,16 +3,266 @@
+    - Have access to a notebook (e.g. ``jupyter notebook``, ``jupyter lab``, etc.)
+    - Be able to access data from your notebook
+    - Nothing else. Unlike most how-to guides, these instructions do *not* assume that you have configured a Data Context by running ``great_expectations init``.
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+Steps
+-----
 
-If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.
+1. **Import sqlalchemy, SqlAlchemyDataset and MetaSqlAlchemyDataset**
 
-If you want to be a real hero, we'd welcome a pull request. Please see :ref:`the Contributing tutorial <tutorials__contributing>` and :ref:`How to write a how to guide` to get started.
+    .. code-block:: python
+
+        import sqlalchemy as sa
+        from great_expectations.dataset import (
+            SqlAlchemyDataset,
+            MetaSqlAlchemyDataset,
+        )
+
+    ``SqlAlchemyDataset`` is the parent class used for executing Expectations using sqlalchemy. Most of the core Expectations are built using decorators defined in ``MetaSqlAlchemyDataset``. These decorators streamline the task of extending Great Expectations with custom Expectation logic.
+
+2. **Define a class inheriting from SqlAlchemyDataset**
+
+    .. code-block:: python
+
+        class CustomSqlAlchemyDataset(SqlAlchemyDataset):
+
+            _data_asset_type = "CustomSqlAlchemyDataset"
+
+    Setting the ``_data_asset_type`` is not strictly necessary, but can be helpful for tracking the lineage of instantiated Expectations and :ref:`Validation Results`.
+
+3. **Within your new class, define Expectations using decorators from MetaSqlAlchemyDataset**
+
+    ``column_map_expectations`` are Expectations that are applied to a single column, on a row-by-row basis.
+
+    The ``@MetaSqlAlchemyDataset.column_map_expectation`` decorator wraps a custom function with all the business logic required to turn it into a fully-fledged Expectation. This spares you the hassle of defining required arguments like ``mostly`` and ``result_format``. Your custom function can focus exclusively on the business logic of passing or failing the Expectation.
+
+    In the simplest case, they could be as simple as one-line lambda functions.
+
+    .. code-block:: python
+
+        @MetaSqlAlchemyDataset.column_map_expectation
+        def expect_column_values_to_be_even(self, column):
+            return (sa.column(column) % 2 == 0)
+
+    The business logic for developing SQL queries using ``MetaSqlAlchemyDataset`` deocrators is a little diffeent from ``MetaPandasDataset``. To use the ``column_map_expectation`` decorator, both require that your custom function accept at least two arguments: ``self`` and ``column``. Both create a function API where the user passes a string containing the column name to your Expectation.
```

Comment:
This comparison seems asymmetric in a very Pandas-first way.

Reply:
I agree. Suggestions welcome---I played with this paragraph for a while and came up blank.

Part of the reason is that I think we probably want to refactor some of the explanation of column_map vs column_aggregate vs other _expectation decorators out into the Reference section.

But I'm trying not to touch Reference material in these PRs, to prevent the scope from blowing up (again.)

Can we merge this as is, with the intention to come back to it later?